src/main/java/core/basesyntax/strategy/CodeService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface CodeService {
+    void doOperation(FruitTransaction fruit);
```

Comment:
We do not need Service for Code in this application
```suggestion
public interface TransactionProcessor {
    void processTransaction(FruitTransaction transaction);
```